### PR TITLE
many, many components (I forgot to commit...)

### PR DESCRIPTION
**Pull request by hfrauenkron**
add components for Badges, Card, CloseButton, Header, MainBackground, Searchbar and Title.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 63401
```
Comment (lmachens) [inline .vscode/settings.json]: This file should not be part of the git repository.

File: public/index.html [hunk 2]
```diff
@@ -24,7 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Fira+Sans:900|Roboto:500&display=swap"
```
Comment (lmachens) [inline public/index.html]: Don't forget to add the font to Storybook too

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      : css`
+          display: none;
+        `}
+`;
+
+const StyledCloseMenuIcon = styled.div`
+  ${props =>
+    props.selected
+      ? css`
+          display: show;
```
Comment (lmachens) [inline src/components/Header.js]: I am not sure if `show`exists. You can set it to `block`

File: src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    props.selected
+      ? css`
+          display: none;
+        `
+      : css`
+          display: show;
+        `}
+`;
+
+export default function Header({ selected }) {
```
Comment (lmachens) [inline src/components/Header.js]: Missing prop types

File: src/icons/CloseIcon.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="14"
+      height="14"
+      viewBox="0 0 25.498 26.054"
+    >
+      <g
+        id="Close-Menu_8"
+        data-name="Close-Menu – 8"
```
Comment (lmachens) [inline src/icons/CloseIcon.js]: You can remove unused properties like `id` and `data-name`

File: .vscode/settings.json (hunk 1 above)
Comment (hfrauenkron) [reply .vscode/settings.json]: 🙈

File: src/components/Header.js (hunk 3 above)
Comment (hfrauenkron) [reply src/components/Header.js]: It worked, but changed it to ```block``` to be sure. Thanks!

**Pull request merged**